src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -321,26 +303,77 @@ class TezosPlatformDiscoveryOperations(
     * @return list of attributes as a Future
     */
   def getTableAttributes(tableName: String): Future[Option[List[Attribute]]] = {
-    attributesCache.read.flatMap { entitiesMap =>
-      entitiesMap.get(tableName).map { case (last, attributes) =>
-        if (last + cacheTTL.toMillis > now) {
-          IO.pure(attributes)
-        } else {
-          for {
-            updatedAttributes <- IO.fromFuture(IO(getUpdatedAttributes(tableName, attributes)))
-            _ <- attributesCache.take
-            _ <- attributesCache.put(entitiesMap.updated(tableName, now -> updatedAttributes))
-          } yield updatedAttributes
-        }
-      }.sequence
-    }.unsafeToFuture()
+    caching.getCachingStatus.map { status =>
+      if (status == Finished) {
```

Comment:
I'm not sure how would you want to do that

Reply:
I guess moving this all status checking and cache expiration internally to the `caching` object, and it's always there that the new data would be loaded as needed, hiding everything from the PlatDiscOp service...

It might be less ideal than I'm imagining, I might be missing some detail. I'm only a bit troubled that the Operations class should explicitly drive manually the caching, based on a public status... usually the cache should hide those details from usage

Reply:
ok, I guess you're right but i think it will be bigger change, I propose to do it in the next caching PR